docs/Theming.md

```diff
@@ -65,65 +63,71 @@ import {
     TextInput,
 } from 'react-admin';
 import Icon from '@material-ui/icons/Person';
-import { withStyles } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core/styles';
 
 export const VisitorIcon = Icon;
 
 // The Filter component supports the `form` and `button` CSS classes. Here we override the `form` class
-const filterStyles = {
+const useStyles = makeStyles({
     form: {
         backgroundColor: 'Lavender',
     },
-};
+});
 
-const VisitorFilter = withStyles(filterStyles)(({ classes, ...props }) => (
-    <Filter classes={classes} {...props}>
-        <TextInput
-            className={classes.searchInput}
-            label="pos.search"
-            source="q"
-            alwaysOn
-        />
-        <DateInput source="last_seen_gte" />
-        <NullableBooleanInput source="has_ordered" />
-        <NullableBooleanInput source="has_newsletter" defaultValue />
-    </Filter>
-));
+const VisitorFilter = props => {
+    const classes = useStyles();
+    return (
+        <Filter classes={classes} {...props}>
+            <TextInput
+                className={classes.searchInput}
+                label="pos.search"
+                source="q"
+                alwaysOn
+            />
+            <DateInput source="last_seen_gte" />
+            <NullableBooleanInput source="has_ordered" />
+            <NullableBooleanInput source="has_newsletter" defaultValue />
+        </Filter>
+    );
+};
 
 // The List component supports the `root`, `header`, `actions` and `noResults` CSS classes. Here we override the `header` and `actions` classes
-const listStyles = {
+const useStyles = makeStyles({
```

Comment:
the name `useStyles` was already defined earlier (line 71), this is a syntax error. That's the reason why the style objects weren't named 'styles' in the first place.